internal/cli/flags.go

```diff
@@ -13,13 +13,19 @@ func addTiltfileFlag(cmd *cobra.Command, s *string) {
 	cmd.Flags().StringVarP(s, "file", "f", tiltfile.FileName, "Path to Tiltfile")
 }
 
-func addWebPortFlag(cmd *cobra.Command) {
-	cmd.Flags().IntVar(&webPort, "port", DefaultWebPort, "Port for the Tilt HTTP server. Set to 0 to disable.")
+// For commands that talk to the web server.
+func addAPIServerFlags(cmd *cobra.Command) {
```

Comment:
the naming scheme between this and `addHTTPServerFlags` isn't really consistent--i might expect this command to START the API server? That said it's not a big deal and i'm not sure I have better ideas... `startServerFlags` and `connectServerFlags`?

Reply:
i like that idea! done